db_store/src/settings.rs

```diff
@@ -0,0 +1,28 @@
+use crate::Result;
+use http::Uri;
+use serde::Deserialize;
+use sqlx::{postgres::PgPoolOptions, Pool, Postgres};
+
+#[derive(Debug, Deserialize, Clone)]
+pub struct Settings {
+    /// Max open connections to the database. If absent a default is calculated
+    /// by application code
```

Comment:
it is done this way because there can be multiple processes in an application that each connect to the db and want to have a different number of max connections.  The example is poc_iot_verifier.

Reply:
But this is just for getting meta values out of the db, not generic Postgres access. You’re saying there’s applications that have multiple processes each with different requirements for number of max connections they want to allow for setting and getting state k/v pairs like “last_something_something_processed”?

Reply:
I know the current code is. But the crate is called db_store which implies more general db functions

Reply:
Jeff and I talked this over and came to an understanding.